payloads/v10/gamesdk.ts

```diff
@@ -0,0 +1,308 @@
+/**
+ * Types extracted from https://discord.com/developers/docs/game-sdk/sdk-starter-guide
+ */
+
+import type { Snowflake } from '../../globals';
+import type { LocalizationMap } from '../common';
+
+export interface APIAchievement {
+	/**
+	 * Unique id of the application
+	 */
+	application_id: Snowflake;
+	/**
+	 * Name of this achievement
+	 */
+	name: string;
+	/**
+	 * Localization dictionary for the `name` field
+	 */
+	name_localizations: LocalizationMap | null;
+	/**
+	 * The description of this achievement
+	 */
+	description: string;
+	/**
+	 * Localization dictionary for the `description` field
+	 */
+	description_localizations: keyof LocalizationMap | null;
+	/**
+	 * Whether this achievement is secret
+	 */
+	secret: boolean;
+	/**
+	 * Whether this achievement is secure
+	 */
+	secure: boolean;
+	/**
+	 * Unique id of this achievement
+	 */
+	id: Snowflake;
+	/**
+	 * The achievement icon hash
+	 */
+	icon_hash: string;
+}
+
+export interface APILobby {
+	/**
+	 * Unique id of this lobby
+	 */
+	id: Snowflake;
+	/**
+	 * The type of this lobby (whether it is private or public)
+	 *
+	 * @default Private
+	 */
+	type: LobbyType;
+	/**
+	 * The user id of the lobby owner
+	 */
+	owner_id: Snowflake;
+	/**
+	 * The password of this lobby
+	 */
+	secret: string;
+	/**
+	 * The maximum capacity of this lobby
+	 *
+	 * @default 16
+	 */
+	capacity: number;
+	/**
+	 * Whether this lobby can be joined
+	 */
+	locked: boolean;
+	/**
+	 * Unique id of the application
+	 */
+	application_id: Snowflake;
+	/**
+	 * Metadata for this lobby. key/value pairs with type `string`
+	 *
+	 * @default {}
+	 */
+	metadata: Record<string, string>;
+	/**
+	 * The region of this lobby. Defaults the region of the requesting server's IP address
+	 */
+	region: LobbyRegion;
+}
+
+export enum LobbyRegion {
+	Atlanta = 'atlanta',
+	Brazil = 'brazil',
+	Bucharest = 'bucharest',
+	BuenosAires = 'buenos-aires',
+	Dubai = 'dubai',
+	Finland = 'finland',
+	Hongkong = 'hongkong',
+	India = 'india',
+	Japan = 'japan',
+	Madrid = 'madrid',
+	Milan = 'milan',
+	Montreal = 'montreal',
+	Newark = 'newark',
+	Rotterdam = 'rotterdam',
+	Russia = 'russia',
+	SantaClara = 'santa-clara',
+	Santiago = 'santiago',
+	Seattle = 'seattle',
+	Singapore = 'singapore',
+	SouthKorea = 'south-korea',
+	SouthAfrica = 'southafrica',
+	StPete = 'st-pete',
+	Stockholm = 'stockholm',
+	Sydney = 'sydney',
+	UsCentral = 'us-central',
+	UsEast = 'us-east',
+	UsSouth = 'us-south',
+	UsWest = 'us-west',
+}
```

Comment:
wheres this from?

Reply:
got these from this request (that btw doesn't use the same regions as `/voice/regions`)
![image](https://user-images.githubusercontent.com/79790819/179234906-8f80c611-2164-4190-a272-7ea72e2ecdc3.png)

Reply:
this is probably just all the possible voice regions, /voice/regions only returns the ones u can manually set voice channels to

this list can change at any time so idk, @vladfrangu do u think it should be included? these are the same as the options for channel rtc_region

Reply:
Lets leave these as string because voice regions will come and go and truthfully, an enum of them is not useful